.github/workflows/flatpak-build.yaml

```diff
@@ -0,0 +1,193 @@
+name: Flatpak Build
+on:
+  workflow_call:
+    inputs:
+      arch:
+        default: x86_64
```

Comment:
Add descriptions for each input, they are mandatory per the YAML schema (at least my linter automatically screams at me about them).